server/components/index.js

```diff
@@ -0,0 +1,21 @@
+const { sectorInformation } = require("./sectorInformation");
+const { topPeers } = require("./topPeers");
+const { companyOverview } = require("./companyOverview");
+const { latestNewsInterval } = require("./latestNewsInterval");
```

Comment:
We should probably rename this to `latestNews` instead of `latestNewsInterval`

Reply:
Or more verb-esque like `getLatestNews`?

Reply:
Would it be better to rename each function to `get......`?

Reply:
Depends if it returns it or not.
`latestNews` is a noun, which indicates an array of news
`getLatestNews` is a function, which returns an array of news.

Your services are neither of these. `emitLatestNews` might be better named?

Reply:
I agree `emit +...` is a better name, will change it

Reply:
Agreed - change to `emitLatestNews` in refactor PR